firmware/quintuna/VC/src/app/states/app_bmsOnState.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "app_stateMachine.h"
+
+/**
+ * Get a pointer to the Drive State.
+ * @return A pointer to the Drive State. THIS SHOULD NOT BE MODIFIED.
+ */
+const State *app_bmsOnState_get(void);
```

Comment:
just extern the state variable out

as long as it's constant we should be good